### PR TITLE
update change log for KeyVault track 2 mgmt sdk

### PR DESCRIPTION
**Pull request by erich-wang**

### CODE REVIEW

File: sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md [hunk 1]
```diff
@@ -2,5 +2,75 @@
...
+The package name has been changed from `Microsoft.Azure.Management.KeyVault` to `Azure.Management.KeyVault`
+
+#### Create Key Vault
+
+Before
+```csharp
+using Microsoft.Azure.Management.KeyVault;
+using Microsoft.Azure.Management.KeyVault.Models;
+
+var keyVaultManagementClient = new KeyVaultManagementClient(credentials);
```
Comment (nickzhums) [inline sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md:25]: should we add how we initialize credentials? 

eg. var credentials = DefaultAzureCredentials()

File: sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md [hunk 2]
```diff
@@ -2,5 +2,75 @@
 
 ## 1.0.0-preview.1
 
-### New Features
-- Initial preview of Azure Management KeyVault SDK based on Azure.Core
\ No newline at end of file
+This is public preview version based on Azure Core. This version uses a next-generation code generator that introduces important new features.
```
Comment (nickzhums) [inline sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md:5]: This package follows the new Azure SDK guidelines which provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more.

This is a Public Preview version, so expect incompatible changes in subsequent releases as we improve the product.  To provide feedback, please submit an issue in our Azure SDK for .NET GitHub repo.

File: sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md [hunk 3]
```diff
@@ -2,5 +2,75 @@
...
+
+### General New Features
+
+    - Support MSAL.NET, Azure.Identity is out of box for supporting MSAL.NET
```
Comment (nickzhums) [inline sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md:9]: For more information about unified authentication, please refer to Azure Identity documentation for .NET (https://docs.microsoft.com//dotnet/api/overview/azure/identity-readme?view=azure-dotnet)

File: sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md [hunk 4]
```diff
@@ -2,5 +2,75 @@
...
+    - Support [OpenTelemetry](https://opentelemetry.io/) for distributed tracing
+    - Support uniform telemetry across all languages
+
```
Comment (nickzhums) [inline sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md:12]: Make a note here:

Note: For more information about unified authentication, please refer to Azure Identity documentation for .NET (https://docs.microsoft.com//dotnet/api/overview/azure/identity-readme?view=azure-dotnet)

File: sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md (hunk 1 above)
Comment (nickzhums) [inline sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md:25]: should we also show how we construct the credentials previously ?

File: sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md [hunk 5]
```diff
@@ -2,5 +2,75 @@
...
 
-### New Features
-- Initial preview of Azure Management KeyVault SDK based on Azure.Core
\ No newline at end of file
+This is public preview version based on Azure Core. This version uses a next-generation code generator that introduces important new features.
+
+### General New Features
+
+    - Support MSAL.NET, Azure.Identity is out of box for supporting MSAL.NET
+    - Support [OpenTelemetry](https://opentelemetry.io/) for distributed tracing
```
Comment (nickzhums) [inline sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md:10]: Add

-HTTP Pipeline with custom policies
-Error-handling that matches language ecosystem

File: sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md [hunk 6]
```diff
@@ -2,5 +2,75 @@
...
+    - Support uniform telemetry across all languages
+
+### Migration from Previous Version of Azure Management SDK
+
+#### Package Name
+The package name has been changed from `Microsoft.Azure.Management.KeyVault` to `Azure.Management.KeyVault`
+
+#### Create Key Vault
+
+Before
```
Comment (nickzhums) [inline sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md:20]: Previous Version:

File: sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md [hunk 7]
```diff
@@ -2,5 +2,75 @@
...
+var keyVaultManagementClient = new KeyVaultManagementClient(credentials);
+var vault = await keyVaultManagementClient.Vaults.BeginCreateOrUpdateAsync
+                (
+                    resourceGroupName,
+                    vaultName,
+                    parameters
+                );
+```
+
+After
```
Comment (nickzhums) [inline sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md:34]: This Version:

File: sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md [hunk 8]
```diff
@@ -2,5 +2,75 @@
...
+    - Support MSAL.NET, Azure.Identity is out of box for supporting MSAL.NET
+    - Support [OpenTelemetry](https://opentelemetry.io/) for distributed tracing
+    - Support uniform telemetry across all languages
+
+### Migration from Previous Version of Azure Management SDK
+
+#### Package Name
+The package name has been changed from `Microsoft.Azure.Management.KeyVault` to `Azure.Management.KeyVault`
+
+#### Create Key Vault
```
Comment (nickzhums) [inline sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md:18]: Management Client Changes

File: sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md [hunk 9]
```diff
@@ -2,5 +2,75 @@
...
+var vault = await vaultsClient.StartCreateOrUpdateAsync(
+            resourceGroupName,
+            vaultName,
+            parameters
+            );
+var vaultValue = (await vault.WaitForCompletionAsync()).Value;
+
+```
+
+#### Create Model Permissions
```
Comment (nickzhums) [inline sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md:54]: Object Model Changes
Example: Create a Permission Model

File: sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md [hunk 10]
```diff
@@ -2,5 +2,75 @@
...
+
+Before
```
Comment (nickzhums) [inline sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md:56]: Before upgrade:

File: sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md [hunk 11]
```diff
@@ -2,5 +2,75 @@
...
+var permissions = new Permissions
+                {
+                    Keys = new string[] { "all" },
+                    Secrets = new string[] { "all" },
+                    Certificates = new string[] { "all" },
+                    Storage = new string[] { "all" },
+                }
+```
+
+After
```
Comment (nickzhums) [inline sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md:67]: After upgrade:

File: sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md (hunk 2 above)
Comment (erich-wang) [reply sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md:5]: Will update with sdk repo url

File: sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md [hunk 12]
```diff
@@ -2,5 +2,87 @@
...
+> NOTE: For more information about unified authentication, please refer to [Azure Identity documentation for .NET](https://docs.microsoft.com//dotnet/api/overview/azure/identity-readme?view=azure-dotnet)
+
+### Migration from Previous Version of Azure Management SDK
+
+#### Package Name
+The package name has been changed from `Microsoft.Azure.Management.KeyVault` to `Azure.Management.KeyVault`
+
+#### Management Client Changes
+
+Example: Create a Key Vault Instace:
```
Comment (isra-fel) [inline sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md:26]:
```suggestion
Example: Create a Key Vault Instance:
```

**Review by nickzhums — APPROVED** [review]
Looks good. Thanks Erich

File: sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md [hunk 13]
```diff
@@ -2,5 +2,87 @@
...
+                );
+```
+
+After upgrade:
+```csharp
+using Azure.Identity;
+using Azure.Management.KeyVault;
+using Azure.Management.KeyVault.Models;
+
+var keyVaultManagementClient = new KeyVaultManagementClient(SubscriptionId,
```
Comment (allenjzhang) [inline sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md:50]: nit: SubscriptionId on separate line

File: sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md [hunk 14]
```diff
@@ -2,5 +2,87 @@
...
+                    Certificates = new string[] { "all" },
+                    Storage = new string[] { "all" },
+                }
+```
+
+After upgrade:
+```csharp
+var permissions = new Permissions
+            {
+                Keys = new KeyPermissions[] { new KeyPermissions("all") },
```
Comment (allenjzhang) [inline sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md:83]: nit: I think `keys = new [] { }` should be sufficient.

File: sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md [hunk 15]
```diff
@@ -2,5 +2,87 @@
...
+                Secrets = new SecretPermissions[] { new SecretPermissions("all") },
```
Comment (allenjzhang) [inline sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md:84]: also, does KeyPremissions has a constant value declared for "all" btw?

Comment (erich-wang) [reply sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md:84]: Unfortunately no for "all", there're constant values for individuals, e.g.  SecretPermissions.Purge, SecretPermissions.List

**Pull request merged**